### PR TITLE
LPS 134692 Info panel collapsible

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
The info panel in the Content Dashboard contains a lot of information and currently it is all being displayed by default. With this story our goal is to make the Categorization and Details section collapsible.
[Link to story](https://issues.liferay.com/browse/LPS-134692)

Proposed Solution
========
Use ClayPanel component to make the sections collapsables.

Steps to Verify:
----------------
1. Create at least one document and one basic web content
2. Assign some categories to one of the elements created in step 1
3. Go to the Content Dashboard and click in the info icon for the contents just created and check that the info panel appears with the collapsable sections.

Change summary (optional):
---------------
* LPS-134692 Make sections collapsables in the info panel
* LPS-134692 Update tests snapshots
* LPS-134692 Refactor of SidebarPanelInfoView component adding new SidebarPanelCollapsibleSection component
* LPS-134692 Categories section expanded by default
* LPS-134692 Move SidebarPanelInfoView components to SidebarPanelInfoView folder

### CODE REVIEW

**Comment by claraizquierdo** [comment]
ci:test:sf

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js [hunk 1]
```diff
@@ -180,93 +182,137 @@ const SidebarPanelInfoView = ({
...
-						<h5 className="c-mb-1 font-weight-semi-bold">
-							{Liferay.Language.get('extension')}
-						</h5>
-
-						<p className="text-secondary">{extension}</p>
+				<ClayPanel.Group className="panel-group-flush panel-group-sm">
+					{showTaxonomies && (
+						<ClayPanel
+							collapsable
+							displayTitle={
```
Comment (beltranrengifo) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js:189]: Maybe we can move all this HTML to a new component, inside this file?

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js [hunk 2]
```diff
@@ -180,93 +182,137 @@ const SidebarPanelInfoView = ({
...
+									</div>
+								)}
+							</ClayPanel.Body>
+						</ClayPanel>
+					)}
 
-						<h5 className="c-mb-1 font-weight-semi-bold">
-							{Liferay.Language.get('size')}
-						</h5>
+					<ClayPanel
```
Comment (beltranrengifo) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js:252]: And maybe in the same way than the previous comment ¿could be a good idea to move all the ClayPanel to another component? We could use `children`prop for the content slot, but the rest is the same in all cases...
What do you think?

**Review by beltranrengifo — COMMENTED** [review]
Hi @claraizquierdo 
I commented some thought, let me know
Thxs!

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js (hunk 2 above)
Comment (claraizquierdo) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js:252]: Hey Beltran, I think that it makes sense to move the Panel to a new Component 👍

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:test:tango

**Review by beltranrengifo — COMMENTED** [review]
Hi @claraizquierdo !
I like it, the JSX in `SidebarPanelInfoView` is shorted and more manageable.

I also propose to create a folder inside `Components`, called `SidebarPanelInfoView` with all the components that are coupled to this one:
- DocumentPreview
- FileUrlCopyButton
- ItemLanguages
- SidebarPanelInfoCollapsibleSection

This way we can avoid to have an unstructured and messy components folder. Do you like it?

Another thing I wonder:
The first collapsable should not be uncollapsed on first load?

Thanks! 🙂

**Comment by claraizquierdo** [comment]
> 

Ey Beltran, I gonna have a look at what you said about the folder. 

About what sections should be uncollapsed on first load, I going to ask Julia Molano about it wasn't clear.

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js [hunk 3]
```diff
@@ -230,6 +231,7 @@ const SidebarPanelInfoView = ({
 					)}
 
 					<SidebarPanelInfoCollapsibleSection
+						expanded={false}
```
Comment (beltranrengifo) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js:234]: Is this prop needed? It's set false by default. Do you prefer to declare it for readability or something like that?

**Review by beltranrengifo — COMMENTED** [review]
Hi @claraizquierdo !
Just a small comment here 👇

**Review by beltranrengifo — APPROVED** [review]
🎉

**Comment by claraizquierdo** [comment]
ci:forward

**Pull request closed without merging**